stable/nginx-ingress/values.yaml

```diff
@@ -81,7 +81,15 @@ controller:
   scope:
     enabled: false
     namespace: ""   # defaults to .Release.Namespace
-
+  
```

Comment:
Linting is failing for this file. Please review the errors.

Reply:
should be ok now